tests/run/worktree/data/simple/testplan.fmf

```diff
@@ -0,0 +1,24 @@
+prepare:
+    script: echo CHECK testfile
+
+execute:
+    script: grep CHECK testfile
+
+finish:
+    script: grep CHECK testfile
+
+/local:
+    provision:
+        how: local
+
+# /minute:
```

Comment:
Let's add the new test to the `tier 1` set. It will then be executed as part of the `/plans/provision/*` plans.